frontend/src/stores/chatStore.ts

```diff
@@ -37,6 +52,55 @@ export const useChatStore = defineStore('chats', {
 			}
 		},
 
+		async createNewChannel(createNewChannelForm: CreateNewChannelForm) {
+			try {
+				this.errors.length = 0;
+				const all = (await getRequest('chats')).data;
+				const found = all.find(
+					(chat: Chat_List_Item) => chat.name === createNewChannelForm.name,
+				);
+				if (found !== undefined){
+					this.errors.push('channel name already exists');
+					return;
+				}
+				if (!createNewChannelForm.name ){
+					this.errors.push('Not a valid channel name');
+					return;
+				}
```

Comment:
can you add `createNewChannelForm.name.trim().length === 0`?
For when someone adds a name with only spaces...

Reply:
like this?

if (found !== undefined && createNewChannelForm.name.trim().length === 0)